js/MetricHandler.js

```diff
@@ -316,7 +316,7 @@ export class MetricHandler {
   loadedMetric (metricBase, metricTraces, metricIndex) {
     const myMetric = this.store.state.allMetrics[metricBase]
     if (!myMetric) {
-      this.store.setMetric(metricBase, new Metric(this.renderer, metricBase, metricTraces))
+      this.store.setMetric(metricBase, new Metric(this.renderer, metricBase, undefined, []))
```

Comment:
Why is `metircTraces` gone here?

I don't know what it is, but it looks like it might have been accidentally replaced by `[]`.